localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```diff
@@ -6,14 +6,181 @@
 #############################
```

Comment:
Dot in title name.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/141678216-588d7a87-7750-46e3-ba90-579a2cc8812f.png)

</details>